core/vm/logger.go

```diff
@@ -29,10 +29,16 @@ import (
 // Note that reference types are actual VM data structures; make copies
 // if you need to retain them beyond the current call.
 type EVMLogger interface {
+	// Transaction level
+	CaptureTxStart(gasLimit uint64)
```

Comment:
Well, I guess one thing though. A tracer/logger which implements this, but not others -- wouldn't they also want to tx hash and index?

Reply:
I had a look and the right now the only tracer taking this info is jsTracer and that through its constructor (which makes it a bit special).

Do you think it'll be useful to add this info to the logger interface? One advantage I can see it would make the js tracer a bit more standard.

Reply:
Well, I do think it would "make sense" to include it here and now: if we're adding tx level hooks, we should also provide the basic tx info. However, I now looked into it a bit more, and the hash/index is not information which we have readily available in `state_transition.go` at this point, so maybe it's more hassle than it's worth... ?